tests/acceptance/package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "@currents/playwright": "^1.5.11",
     "@playwright/test": "1.45.3",
-    "@shopware-ag/acceptance-test-suite": "^9.0.0",
+    "@shopware-ag/acceptance-test-suite": "^10.2.0",
```

Comment:
in future avoid bumping dependencies while adding tests, that should be a separate PR

related PRs that are doing bump and you will conflict each other: #5931 #5865